scripts/issue/355/set_new_sett_controllers.py

```diff
@@ -0,0 +1,109 @@
+import json
+import os
+import web3
+
+from brownie import accounts, chain, Contract, interface, web3
+from eth_abi import encode_abi
+from rich.console import Console
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import ADDRESSES_ETH, registry
+
+
+console = Console()
+
+# Get addresses
+DEV_MULTI = registry.eth.badger_wallets.dev_multisig
+NATIVE_CONTROLLER = registry.eth.controllers.native
+
+KEYS = [
+    "bbveCVX-CVX-f",
+    "bcrvBadger",
+    "bcrvIbBTC",
+]
+
+STRATEGIES = {
+    "bbveCVX-CVX-f": ADDRESSES_ETH["strategies"]["native.bbveCVX-CVX-f"],
+    "bcrvBadger": ADDRESSES_ETH["strategies"]["native.bcrvBadger"],
+    "bcrvIbBTC": ADDRESSES_ETH["strategies"]["native.bcrvIbBTC"],
+}
+
+VAULTS = {
+    "bbveCVX-CVX-f": ADDRESSES_ETH["sett_vaults"]["bbveCVX-CVX-f"],
+    "bcrvBadger": ADDRESSES_ETH["sett_vaults"]["bcrvBadger"],
+    "bcrvIbBTC": ADDRESSES_ETH["sett_vaults"]["bcrvIbBTC"],
+}
+
+# Set up safe
+safe = GreatApeSafe(DEV_MULTI)
+safe.init_badger()
+
+# Other constants
+TX_DIR = "data/badger/timelock/upgrade_new_sett_controllers/"
+
+
+def main(submitTx="true", queue="true", simulation="true"):
```

Comment:
Imo, the simulation should be written in such a way that allows to review the behavior of the whole process sucessful execution. Aka, when running the simulation, the script should emulate the timelock txs execution (You can impersonate the timelock for this) and then ensure that, once executed, regular operations will work as expected - as you are doing here.

In this case, you can only simulate once the txs are queued and the timelock period executed. Also, it would be good is the safe doesn't post when running the simulation, it is just cleaner and safer.

You can refer to this upgrade script: https://github.com/Badger-Finance/badger-multisig/pull/523/files

Finally, adding the commands with the different combination of possible flags to the PR description would be very handy to guide reviewers around their intended behavior.